sessions/2018/november/session-18.md

```diff
@@ -0,0 +1,198 @@
+# SOFTWARE ENGINEERING IN JAVA
+
+## Session 18 (27/11/2018)
+
+- Reviewing homework
+- Adding BugSnag to my API code
+- Partial Git commits
+
+### Notes
+
+#### Reviewing my Homework
+
+Roger was reviewing my homework and made me several suggestions that has to be addressed:
+
+- If I go to <http://localhost:8080/api/factorial/30>, I receive the result `265252859812191058636308480000000`. But If use the frontend this is rounding the result to `2.6525285981219107e+32`. So this must also show the numbers with the same precision as in the backend.
+
+- Verify numbers are being showed as numbers and not as strings.
```

Comment:
I think it's easier if we send the numbers as **`String`s** from the backend, and then just render them as-is on the frontend. This way we don't have to worry about JavaScript's imprecise `Number` type.